content/en/docs/guides/traffic_management/rate_limiting.md

```diff
@@ -11,17 +11,21 @@ Rate limiting is an effective mechanism to control the throughput of traffic des
 
 Most commonly, when a large number of clients are sending traffic to a target host, if the target host becomes backed up, the downstream clients will overwhelm the upstream target host. In this scenario it is extremely difficult to configure a tight enough circuit breaking limit on each downstream host such that the system will operate normally during typical request patterns but still prevent cascading failure when the system starts to fail. In such scenarios, rate limiting traffic to the target host is effective.
 
-OSM supports server-side rate limiting per target host, also referred to as `local per-instance rate limiting`.
+OSM supports two forms of rate limiting:
+1. `Local per-instance rate limiting` at each upstream host
+2. `Global rate limiting` using a global gRPC rate limiting service
 
-## Configuring local per-instance rate limiting
-
-OSM leverages its [UpstreamTrafficSetting API][1] to configure rate limiting attributes for traffic directed to an upstream service. We use the term `upstream service` to refer to a service that receives connections and requests from clients and return responses. The specification enables configuring local rate limiting attributes for an upstream service at the connection and request level. OSM leverages [Envoy's local rate limiting functionality](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/network_filters/local_rate_limit_filter#config-network-filters-local-rate-limit) to implement per-instance local rate limiting at each upstream host.
+OSM leverages its [UpstreamTrafficSetting API][1] to configure rate limiting attributes for traffic directed to an upstream service. We use the term `upstream service` to refer to a service that receives connections and requests from clients and return responses.
```

Comment:
Why use a footnote instead of a hyperlink here?

Reply:
Are you referring to the usage of `[1]` instead of the hyperlink? If yes, did this to primarily avoid repeating the same link multiple times in writing as it's used several times in this doc.

Reply:
Gotcha. I'd say that's probably fine with the caveat that it's likely harder to navigate to the reference. Maybe hyperlink it the first time? Either way, it's not blocking

Reply:
Fixed. To my surprise, it was only referenced once, so changed this to a relative link instead.